### ISSUE TITLE
Issue on page /notebooks/quick_start.html

### ISSUE THREAD

**Issue by Neurosc** [DeepLearning-JupyterBook/deeplearning-jupyterbook.github.io#22]
"In this example, we define a custom dataloader class, ShapeDataset, by inheriting from torch.utils.data.Dataset. 

This sentence is confusing to me.  I feel like it needs an extra sentence to what this is. maybe something like this: 

...By filling in these required functions, we teach PyTorch exactly how to access and load our specific data. Custom datasets in PyTorch should implement two main functions:"

**Comment by Neurosc** [comment]
Now that we have created our dataset, we need to create PyTorch dataloaders to efficiently manage and load our data during model training and evaluation. PyTorch dataloaders make it easier to handle large datasets by loading small batches of data sequentially, which is essential for efficient memory usage and faster processing.

After this sentence, it would be good to precisely define what dataloaders are. It was confusing to me at first. Maybe something like this?

A DataLoader is what actually:

Takes your Dataset
Loads data in batches (not one at a time)
Shuffles the data if you want
Handles parallel loading
Feeds batches to your neural network during training
